### PR TITLE
fix(pacstall-gui-git): Change `settings.sh` install location

### PR DESCRIPTION
**Pull request by cat-master21**
In my script, the location of `settings.sh` has changed so I have updated this to reflect that.

### CODE REVIEW

File: packages/pacstall-gui-git/pacstall-gui-git.pacscript [hunk 1]
```diff
@@ -12,17 +12,18 @@ pkgver() {
 }
 
 prepare() {
-  sudo mkdir -p "${pkgdir}/usr/bin" "${pkgdir}/usr/lib/pacstall-gui" "${pkgdir}/usr/share/applications"
+  sudo mkdir -p "${pkgdir}/usr/bin" "${pkgdir}$HOME/.config/pacstall-gui" "${pkgdir}/usr/share/applications"
```
Comment (Elsie19) [inline packages/pacstall-gui-git/pacstall-gui-git.pacscript:15]: Don't use `$HOME`, use `$homedir`.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — CHANGES_REQUESTED** [review]
Actually where should we put these files, because we shouldn't be packing a Deb with hardcoded paths to home directories.

**Comment by cat-master21** [comment]
In that case @Elsie19 I can just move it to the user's home directory. It is okay my program automatically regenerates the file in case it is missing.

File: packages/pacstall-gui-git/pacstall-gui-git.pacscript [hunk 2]
```diff
@@ -12,17 +12,18 @@ pkgver() {
 }
 
 prepare() {
-  sudo mkdir -p "${pkgdir}/usr/bin" "${pkgdir}/usr/lib/pacstall-gui" "${pkgdir}/usr/share/applications"
+  sudo mkdir -p "${pkgdir}/usr/bin" "$homedir/.config/pacstall-gui" "${pkgdir}/usr/share/applications"
```
Comment (Elsie19) [inline packages/pacstall-gui-git/pacstall-gui-git.pacscript:15]: Do not put `$homedir` anything in any functions except post ones.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/pacstall-gui-git/pacstall-gui-git.pacscript (hunk 2 above)
Comment (cat-master21) [reply packages/pacstall-gui-git/pacstall-gui-git.pacscript:15]: Is the `settings.sh` file still available in post install if it is not put anywhere else though?

**Review by Elsie19 — COMMENTED** [review]
Does this PR do anything? It seems to just be removing the creation of `"${pkgdir}/usr/lib/pacstall-gui"`, and removing `settings.sh`entirely.

**Comment by cat-master21** [comment]
Yes, as in the script, `settings.sh`'s location got moved to `~/.config/pacstall-gui` but is not really needed because the script generates it if it is missing but it is not needed at `/usr/lib/pacstall-gui` anymore @Elsie19.

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**